test-app/README.md

```diff
@@ -9,15 +9,15 @@ You will need the following things properly installed on your computer.
 
 * [Git](https://git-scm.com/)
 * [Node.js](https://nodejs.org/)
-* [Yarn](https://yarnpkg.com/)
+* [PNPm](https://pnpm.io/)
```

Comment:
Is this a typo or m should be lower case?

Reply:
It's a typo. But now I wonder how we want to write it: `PNPM`, `pnpm` or `Pnpm`?

Reply:
I would go for `pnpm` the same way we usually write `npm`, lower case is used on https://pnpm.io/ title and docs.

Reply:
Updated with `pnpm`. Thanks @BlueCutOfficial!

Reply:
Actually, on https://pnpm.io the title is different every time. So it looks like any of the formats is correct? 😅